src/application/cli.py

```
@@ -0,0 +1,205 @@
+import asyncio
+import argparse
+from pathlib import Path
+from datetime import datetime
+import sys
+from typing import List, Optional
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)